judge/models/problem.py

```diff
@@ -5,7 +5,7 @@
 from django.core.cache import cache
 from django.core.validators import RegexValidator
 from django.db import models
-from django.db.models import F, QuerySet
+from django.db.models import F, QuerySet, CASCADE, SET_NULL
```

Comment:
On [line 363](https://github.com/DMOJ/site/blob/a856120187627ce046567f1c8ddbbac0872a189c/judge/models/problem.py#L363), the `models.SET_NULL` should just be `SET_NULL`.